app/components/provider_interface/conditions_list_component.rb

```diff
@@ -3,7 +3,13 @@ class ConditionsListComponent < ViewComponent::Base
     attr_reader :conditions
 
     def initialize(conditions)
-      @conditions = conditions
+      @conditions = sorted_conditions(conditions)
+    end
+
+  private
+
+    def sorted_conditions(conditions)
+      conditions.sort_by { |c| [OfferCondition::STANDARD_CONDITIONS.index(c.text) || Float::INFINITY, c.created_at] }
```

Comment:
can we do this when retrieving the conditions from the database instead, on the `OfferConditions` model.

Reply:
Because the component list is used in the create/change offer flow, there are newly created conditions in the wizard that aren't the result of a db query.